src/ui/page/subscriptions/view.jsx

```diff
@@ -1,53 +1,78 @@
 // @flow
-import React, { useEffect, useState } from 'react';
+import * as PAGES from 'constants/pages';
+import React, { useEffect } from 'react';
 import Page from 'component/page';
 import FileList from 'component/fileList';
 import Button from 'component/button';
 
 type Props = {
-  subscribedChannels: Array<string>, // The channels a user is subscribed to
-  subscriptionContent: Array<{ uri: string, ...StreamClaim }>,
+  subscribedChannels: Array<{ uri: string }>, // The channels a user is subscribed to
   suggestedSubscriptions: Array<{ uri: string }>,
   loading: boolean,
   doFetchMySubscriptions: () => void,
   doFetchRecommendedSubscriptions: () => void,
+  location: { search: string },
+  history: { push: string => void },
+  doClaimSearch: (number, {}) => void,
+  uris: Array<string>,
 };
 
 export default function SubscriptionsPage(props: Props) {
   const {
-    subscriptionContent,
     subscribedChannels,
     doFetchMySubscriptions,
     doFetchRecommendedSubscriptions,
     suggestedSubscriptions,
     loading,
+    location,
+    history,
+    doClaimSearch,
+    uris,
   } = props;
+
   const hasSubscriptions = !!subscribedChannels.length;
-  const [showSuggested, setShowSuggested] = useState(!hasSubscriptions);
+  const { search } = location;
+  const urlParams = new URLSearchParams(search);
+  const viewingSuggestedSubs = urlParams.get('view');
+
+  function onClick() {
+    let url = `/$/${PAGES.SUBSCRIPTIONS}`;
+    if (!viewingSuggestedSubs) {
+      url += '?view=discover';
+    }
+
+    history.push(url);
+  }
 
   useEffect(() => {
     doFetchMySubscriptions();
     doFetchRecommendedSubscriptions();
   }, [doFetchMySubscriptions, doFetchRecommendedSubscriptions]);
 
+  const idString = subscribedChannels.map(channel => channel.uri.split('#')[1]).join(',');
```

Comment:
is the channel uri guaranteed to have a `#`? they will never be URIs without a claim id?

Reply:
The channel id is required (for now?) in internal-apis for subscriptions. The app always uses the full uri for the subscribeButton component